pandas/tests/window/test_numba.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import pytest
+
+import pandas.util._test_decorators as td
+
+from pandas import Series
+import pandas.util.testing as tm
+
+
+@td.skip_if_no("numba", "0.46.0")
```

Comment:
can't you use the engine fixture

Reply:
All these tests don't make sense with just the `'cython'` engine.